### ISSUE TITLE
"sameOriginWithAncestors is unused" ?  should s/unused/false/ ?

### ISSUE THREAD

**Issue by equalsJeffH** [w3c/webappsec-credential-management#116]
There is this in [#create-passwordcredential](https://w3c.github.io/webappsec-credential-management/#create-passwordcredential):
> 1. Assert: options["password"] exists, and sameOriginWithAncestors is unused.

And also in [#create-federatedcredential](https://w3c.github.io/webappsec-credential-management/#create-federatedcredential):
> 1. Assert: options["federated"] exists, and sameOriginWithAncestors is unused.

Should "unused" actually be `false` in the above two places?  "unused" does not appear to be a term used in the various whatwg specs (but my search-fu is occasionally not-so-great).

**Comment by equalsJeffH** [comment]
Ah, upon further inspection and reflection, it appears that what the phrase "...`sameOriginWithAncestors` is unused" is asserting is that `sameOriginWithAncestors` is not referenced in the ensuing algorithm(s), i.e., it is ignored. Yes?

**Comment by mikewest** [comment]
Correct. "Assert" is an editorial statement, not a normative one: it's basically explaining what's going on in some way. Is there some way we could say "unused" that would have avoided the confusion?

**Comment by equalsJeffH** [comment]
Now that I've been poking around in web platform specs for a while longer, I'd tend to interpret the statement:
> 1. Assert: options["password"] exists, and sameOriginWithAncestors is unused.

...as saying that the `sameOriginWithAncestors` argument to [PasswordCredential's \[\[Create\]\]() method](https://www.w3.org/TR/credential-management-1/#create-passwordcredential) is unused by the caller, i.e., it is not present...? 

Perhaps the phrase ", and sameOriginWithAncestors is unused" can be deleted because we are not making any decisions based on whether `sameOriginWithAncestors` is present, or has a value...?

in any case this is very low priority it seems.
